src/main/java/com/safetynet/alerts/presentation/controller/BaseController.java

```diff
@@ -0,0 +1,15 @@
+package com.safetynet.alerts.presentation.controller;
+
+import com.safetynet.alerts.logging.LogHandlerTiny;
+import com.safetynet.alerts.presentation.model.SafetyAlertsModel;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class BaseController {
+
+    @Autowired
+    LogHandlerTiny logHandler;
+
+    @Autowired
```

Comment:
I would remote this (`@Autowired SafetyAlertsModel safetyAlertsModel;`) from BaseController and keep it in BaseService. And on the BaseService add a method eg. `bool isDataLoaded()` that can be used by any service that is used in the controller

Reply:
makes sense, done for all controllers